src/Albany_Application.cpp

```diff
@@ -1996,8 +1993,15 @@ Application::computeGlobalTangent(
   if (Teuchos::nonnull(params)) {
     TanFadType p;
     int        num_cols_tot = param_offset + num_cols_p;
```

Comment:
So this, more than a number of columns, now represent the upper bound for the derivatives for this parameter, right? That is, the derivs for this parameter will be at entries `[param_offset,num_cols_tot)` in the Fad, which will _always_ have length equal to the total number of parameters. Correct?

Reply:
yes, I think so.